apiary.apib

```diff
@@ -4589,6 +4589,155 @@ Returns all workspaces for the component configuration.
 
    + Attributes (array[Workspace])
 
+# Group Search
+## Search Tables [/v2/storage/search/tables?metadataKey={metadataKey}&metadataValue={metadataValue}&metadataProvider={metadataProvider}&include={include}]
+
+### Search Tables [GET]
+Search all tables accessible by the token by either metadataKey, metadataValue or metadataProvider. By default, all tables are returned with their
+attributes and information about the containing bucket.
+
++ Parameters
+    + include (optional, enum[string]) - Comma separated list of resources to include for each table.
+        + Members
+            + attributes
+            + buckets
+            + columns
+            + metadata
+            + columnMetadata
+        + Default: attributes,metadata
+    + metadataKey (optional, string) - string to search with exact match on one of existing metadata.key
```

Comment:
ale aspon jedno by teda melo byt povinny, nebo to pak listuje vsechny tabulky v projektu? to crashne na tom, ze ta response nevejde do pameti ne? - takze by tam pak muselo byt strankovani

Reply:
No pokud tam není žádný tak by to nemělo spadnout protože je to pak stejné jako list tables. Každopádně do api clienta přidám aby byl aspoň jeden nastavený, ale apiary nevím jak to udělat aby byl aspoň jeden kterýkoliv poslaný.

Reply:
aha, tak asi dobry

na apiary to zadefinovat afaik nijak nejde, muselo by se to tam napsat

Reply:
No podle testů to je teďka tak, že tomu jde procpat prázdný filt a vylistuje to všechny tabulky:
https://github.com/keboola/storage-api-php-client/pull/340/files#diff-fcd797a1839eb8f9efa1ffee16ff7569R42-R43
Takže to podle mě resolved není.

Reply:
Upravil sem a přidal do apib poznámku